src/screens/MakeClipScreen.tsx

```diff
@@ -402,7 +403,7 @@ export class MakeClipScreen extends React.Component<Props, State> {
                   size={24} />
               </TouchableOpacity>
               <TouchableOpacity
-                onPress={() => togglePlay()}
+                onPress={() => togglePlay(this.global)}
```

Comment:
would you only want to pass the state object that you use in toggle play? I checked and we are only using the playbackRate from the player. maybe pass this.global.player.playbackRate only? Or even just the player. instead of passing the whole global state object?

Reply:
Oops yeah I'm trying to avoid passing in the whole this.global everywhere, but haven't swept through the whole app to clean them up.

Actually it turns out that playbackRate param was meaningless though, so I removed it.